helm/alfresco-content-services/tests/deployment-repository_test.yaml

```diff
@@ -17,6 +17,18 @@ tests:
           value: RELEASE-NAME-alfresco-cs-repository
         template: deployment-repository.yaml
 
+  - it: Log4j & Log4j2 test
+    values: *testvalues
+    set:
+      repository.extraLogStatements.testkey: testvalue
```

Comment:
needs to be yaml formatted

Reply:
the volumemount has a subpath set ( to `dev-log4j.properties`) did you check that the resulting pod really has both the `dev-log4j.properties` & `dev-log4j2.properties` mounted?. I would expect it doesn't.